app/helpers/validateSectionData.js

```diff
@@ -24,14 +24,22 @@ export const validateSectionData = (sectionManifest, sectionData) => {
   sectionManifest.questions.map((sectionQuestion) => {
     if (sectionData[sectionQuestion.id] && sectionQuestion.saveValidations) {
       sectionQuestion.saveValidations.map((saveValidation) => {
-        sectionData[sectionQuestion.id].map((sectionDataField, sectionDataFieldId) => {
-          if (validationRules[saveValidation.type].rule(sectionDataField, saveValidation)) {
-            const error = createErrorForField(
-              sectionQuestion.id, sectionDataFieldId, saveValidation.message,
-            );
-            validationErrors.push(error);
-          }
-        });
+        if (sectionQuestion.type === 'bulletpoint-list') {
+          sectionData[sectionQuestion.id].map((sectionDataField, sectionDataFieldId) => {
+            if (validationRules[saveValidation.type].rule(sectionDataField, saveValidation)) {
+              const error = createErrorForField(
+                sectionQuestion.id, sectionDataFieldId, saveValidation.message,
+              );
+              validationErrors.push(error);
+            }
+          });
+        } else if (validationRules[saveValidation.type]
+          .rule(sectionData[sectionQuestion.id], saveValidation)) {
```

Comment:
Null reference if no validation rule exists for the type

Reply:
added a guard to check there is a validation rule before executing the rule and added a test to check this works as expected